polaris-react/src/components/SkeletonPage/SkeletonPage.tsx

```diff
@@ -32,6 +32,13 @@ export function SkeletonPage({
 }: SkeletonPageProps) {
   const i18n = useI18n();
 
+  useEffect(() => {
+    // performance mark on skeleton unmount to help with page load tracking
+    return () => {
+      performance?.mark?.('polaris:page_skeleton:unmount');
```

Comment:
also since this is external I just didnt know if this could get pulled in by an old version of node for SSR and die if the performance object wasn't there.